integration-tests/startup.spec.js

```diff
@@ -9,153 +9,167 @@ const {
 const path = require('path')
 const { assert } = require('chai')
 
-describe('startup', () => {
-  let agent
-  let proc
-  let sandbox
-  let cwd
-  let startupTestFile
-
-  before(async () => {
-    sandbox = await createSandbox()
-    cwd = sandbox.folder
-    startupTestFile = path.join(cwd, 'startup/index.js')
-  })
-
-  after(async () => {
-    await sandbox.remove()
-  })
-
-  context('programmatic', () => {
-    beforeEach(async () => {
-      agent = await new FakeAgent().start()
+const execArgvs = [
+  [],
+  ['--import', 'dd-trace/register.js'],
```

Comment:
The `register(...)` function didn't exist until v20.6 so we'll need some version detection here to only use this on newer versions.

Reply:
Version checks added